xwiki/content.md

```diff
@@ -17,8 +17,6 @@ The goal is to provide a production-ready XWiki system running in Docker. This i
 
 # How to use this image
 
-[![](https://images.microbadger.com/badges/version/xwiki/xwiki-mysql-tomcat.svg)](https://microbadger.com/images/xwiki/xwiki-mysql-tomcat) [![](https://images.microbadger.com/badges/image/xwiki/xwiki-mysql-tomcat.svg)](https://microbadger.com/images/xwiki/xwiki-mysql-tomcat)
-
```

Comment:
ok I'll need to remember to remove those when in the future I'll sync the README from https://github.com/xwiki-contrib/docker-xwiki with the official image README! :)